### ISSUE TITLE
Keyboard accessibility of filter assistant, open/close more/less features, and info 

### ISSUE THREAD

**Issue by lakeen** [w3c/wai-eval-tools#83]
While trying to navigate the tools list using my keyboard, I noticed that the filter assistant is not in the tab order and does not appear to be keyboard accessible.

The same is true for the open/close, more/less, and info icon in the filter list.

**Comment by MicheleAWilliams-A11y** [comment]
Was going to note this as well - the page is missing significant keyboard accessibility in the Search column. There are no focusable elements, only SVG's and text.

**Comment by carlosapaduarte** [comment]
Was also going to report this. It is impossible for a keyboard used to select options from the "Operating system", "Browser", "Output" and "Language" categories because the expand/collapse icon is not keyboard accessible.

**Comment by VeraLange** [comment]
Thank you for your comments. We've implemented keyboard accessibility throughout the resource and a complete accessibility audit is being done at the moment to find other possible issues related to this topic.

**Comment by shawna-slh** [comment]
> complete accessibility audit is being done at the moment to find other possible issues

+1

Let's get the accessibility issues taken care of before bringing it back to EO
